foundation-extensions/mobile-core/configuration/configuration-api-reference.md

```diff
@@ -192,317 +160,367 @@ ACPCore.ConfigureWithAppID("1423ae38-8385-8963-8693-28375403491d");
 {% endtab %}
 {% endtabs %}
 
-## updateConfiguration
-
-You can also update the configuration programmatically by passing configuration keys and values to override the existing configuration.
+## configureWithFileInAssets
 
 {% hint style="info" %}
-Keys that are not found on the current configuration are added when this method is followed. Null values are allowed and replace existing configuration values.
+This API is only available in Android.
```

Comment:
we can bundle the two info blocks together: `This API is only available in Android and was added in Android Core version 1.7.0.`